agent/eventhandler/task_handler.go

```diff
@@ -49,6 +49,16 @@ const (
 	submitStateBackoffMax            = 30 * time.Second
 	submitStateBackoffJitterMultiple = 0.20
 	submitStateBackoffMultiple       = 1.3
+
```

Comment:
Can you provide reasoning behind those values?

Reply:
Do you mean lines 49-51? I'm not sure, because I did not add these values, but I'll remove the empty space in line 52 .

Reply:
Just a note: See https://docs.aws.amazon.com/AmazonECS/latest/APIReference/request-throttling.html for ECS API throttling quota.

Reply:
No I mean the lines you added